### PR TITLE
[v0.4] Set FailurePolicy to Ignore for settings validator

### PR DESCRIPTION
**Pull request by pmatseykanets**
## Issue: 

Backport of #446

## CheckList
  
- [x] Test
  
- [ ] Docs

### CODE REVIEW

**Review by tomleb — APPROVED** [review]

**Review by maxsokolovsky — DISMISSED** [review]

**Review by MbolotSuse — CHANGES_REQUESTED** [review]
Blocking merge until we decide on milestone.

**Review by MbolotSuse — APPROVED** [review]
LGTM please squash when merging.

**Review by ericpromislow — APPROVED** [review]
LG. Same code and test sequence as PR 446.

**Pull request merged**
